libs/types/General.ts

```diff
@@ -15,3 +15,9 @@ export type Result = {
     message?: string;
     error?: string;
 };
+
+export type APIResult = {
+    status: 'error' | 'success';
```

Comment:
did you observe this in one of the responses? I wouldn't have thought anything should have this

Reply:
nah this was just so I could fix it for testing with the least amount of work. A lot of the error handling on the front end uses these status's